honeybee_vtk/_helper.py

```diff
@@ -0,0 +1,28 @@
+"""Helper functions to be used in other modules."""
+
+
+def _check_tuple(val, val_type, max_val=None):
+    """Check if all values in the tuple are selected object type.
+
+    Args:
+        color: User input for color value
+        val_type: Object type that you'd want as values in a tuple. Examples are
+            int or float.
+        max_val: A number either integer or float. The values in the tuple shall be
+            less than this number.
+
+    Returns:
+        A boolean value if True or None.
+    """
+    # Check if all values in tuple are of expected object type
+    item_check = [isinstance(v, val_type) for v in val]
+
+    # Check if all values are less than the maximum allowed value
+    if max_val:
+        val_check = [v < max_val for v in val]
+    else:
+        val_check = [True] * len(val)
+
+    # final check
+    if item_check.count(True) == len(val) and val_check.count(True) == len(val):
+        return True
```

Comment:
I did not want to create this method. I looked at the typing module in Honeybee-core first. See [this](https://github.com/ladybug-tools/honeybee-core/issues/418) issue.
1. I need a stronger type checking
2. I want to return a single boolean value rather than the items.

Reply:
This is not a blocking change so if you like it to stay like this keep it but I can't get behind the logic here.

1. You can just use `map` functionality to do the same.
2. The code never really uses the boolean return. If you use maps it's more flexible and it give you similar results and you use the items that are created in the process.

Reply:
Implemented using the `map` function.